app/src/src/components/common/PickImage.jsx

```diff
@@ -1,36 +1,27 @@
 import * as ImagePicker from 'expo-image-picker';
-
-// Select an image from the camera roll and change the image
-// TODO: can get and change the image on edit profile view, but it doesn't actually save the image when you press save
+import * as ImageManipulator from 'expo-image-manipulator';
+/* Select an image from the camera roll. Returns the local uri to the image. 
+   The local uri can also be used on the page without saving it, so if the "undo changes"
+   or "back" buttons are pressed, then the image isn't saved (unless you press the "save" button) 
+   Quality and compress are both 0 so high quality images can be saved easily.
+   */
 
 const pickImage = async ({ set }) => {
   const result = await ImagePicker.launchImageLibraryAsync({
     mediaTypes: ImagePicker.MediaTypeOptions.Images,
     allowsEditing: true,
-    aspect: [4, 3],
-    quality: 1,
+    aspect: [1, 1],
+    quality: 0,
```

Comment:
I made the quality on the ImagePicker and compress on the ImageManipulator to 0 so there's no error when someone tries to save an image (before I'd get a 413 Request Entity Too Large error). The lower quality isn't really noticeable though.

Reply:
Perhaps the amount to compress will change depending on the platform? Maybe this will fix the issue we had when an android phone uploads an image.

Reply:
I changed the quality and compress values to 0.5 for android and it looks like it has a similar quality as ios now. I tried to make the quality and compress values as 1 for android (and also tried to keep one of them 0 and the other 1) but I got the 413 Request Entity Too Large error again. With the images I used to test it, 0.5 value seems to save fine without errors.